### DISCUSSION TITLE
Unlock door (intercom)- command?

### DISCUSSION THREAD

**Discussion by N3orun** [tsightler/ring-mqtt#637] [Support Q&A]
HI, installed and connected, getting the device info in the log as well. But I seem to be unable to get anything displayed on the Cards besides: Up to date.

Basically I`m just looking for a solution that allows me to use the "Unlock door" trigger in automations. 

Thanks in advance!

**Comment by CarnivoreD** [comment]
I have a similar issue: The topic „Unlock door“ is not available. Is there something to do to activate this topic?

Thank you for any help...
